packages/react-core/src/components/Icon/examples/IconInline.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Icon } from '@patternfly/react-core';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+
+export const IconInline: React.FunctionComponent = () => (
+  <React.Fragment>
+    An inline
+    <Icon isInline>
+      <PlusCircleIcon />
+    </Icon>
+    plus icon.
```

Comment:
It might be good to change the container's font-size and color to show what this does. The core examples use the `<Text>` component.

Reply:
Showing that the inline icon follows text style is why I had put in a few different examples of text.